manifest.json

```diff
@@ -40,7 +40,7 @@
         },
         {
           "sharedTags": {
-            "1.1.5": {},
+            "1.1.6": {},
```

Comment:
We shouldn't bump tag version since we are not actually changing the runtime or the microsoft/dotnet base image. The tag can stay the same because all this is doing is adding more packages to the optimization and restore cache.

Reply:
Interesting, so we version with the runtime instead of our own releases?

Reply:
So far, we have tried to keep the tagging aligned with microsoft/dotnet. If 1.1.6 goes out, this would be the first release without a corresponding base image update.

Reply:
Ah I see, I'll revert to 1.1.5 since there is no runtime update.